chia/cmds/show.py

```diff
@@ -118,8 +118,7 @@ async def show_async(
 
                 host = con["peer_host"]
                 # Strip IPv6 brackets
-                if host[0] == "[":
-                    host = host[1:39]
+                host = host.strip("[]")
```

Comment:
For the record, this instead hazards removing multiple square brackets from either end, such as trailing `]]]`, and the wrong brackets at the wrong end, such as leading `]` or trailing `[`.  I'm not an IPv6 address expert but if those are cases that need to be handled then perhaps we should delegate this to a library meant for handling such things.

Reply:
thanks for keeping my address anonymous 🥳